client/app/account/billing/autoRenew/common/actions-menu.html

```diff
@@ -41,7 +41,7 @@
                 class="btn btn-link"
                 data-ng-class="{ 'disabled' : !canResiliate(service, user) }"
                 data-ng-disabled="!canResiliate(service, user)"
-                data-ng-click="canResiliate(service, user) && resiliateService(service)"
```

Comment:
in case it's not used anymore, you can remove canResiliate method

Reply:
Well... read again 😄